frontend/js/behaviors/core/primaryNavigation.js

```diff
@@ -7,11 +7,29 @@ export default function(container) {
     const menuItemQuery = 'a[role="menuitem"]'
     const canExpandAttribute = 'aria-haspopup'
     const isExpandedAttribute = 'aria-expanded'
+    const exhibitionApiPath = '/api/v1/exhibitions/search'
+    const featuredExhibitionApiQuery = new URLSearchParams({
+        'fields': 'title,image_url,web_url',
+        'query[bool][must][][range][aic_end_at][gte]': 'now',
+        'query[bool][must][][range][aic_start_at][lte]': 'now',
+        'query[bool][must][][term][is_published]': true,
+        'query[bool][must_not][term][status]': 'Closed',
+        'size': 1,
+        'sort': 'position',
+    })
```

Comment:
Remove the status field condition. The "Closed" value is not reliably set.

Remove the `aic_end_at` condition. Occasionally, there can be an ongoing exhibition that doesn't have an end date put into the 0th positions

Reply:
Removed unnecessary conditions